spec/index.html

```diff
@@ -105,189 +110,275 @@ <h2>Abstract</h2>
 
-    <section id="sparql12-documents" class="introductory">
-      <h2>SPARQL 1.2 Documents</h2>
-      <ul>
-        <li><a href="../../sparql-new/spec/index.html">SPARQL 1.2 New</a></li>
-        <li><a href="../../sparql-concepts/spec/index.html">SPARQL 1.2 Overview</a></li>
-        <li><a href="../../sparql-query/spec/index.html">SPARQL 1.2 Query Language</a></li>
-        <li><a href="../../sparql-update/spec/index.html">SPARQL 1.2 Update</a></li>
-        <li><a href="../../sparql-entailment/spec/index.html">SPARQL 1.2 Entailment Regimes</a></li>
-        <li><a href="../../sparql-service-description/spec/index.html">SPARQL 1.2 Service Description</a></li>
-        <li><a href="../../sparql-federated-query/spec/index.html">SPARQL 1.2 Federated Query</a></li>
-        <li><a href="../../sparql-results-json/spec/index.html">SPARQL 1.2 Query Results JSON Format</a></li>
-        <li><a href="../../sparql-results-csv-tsv/spec/index.html">SPARQL 1.2 Query Results CSV and TSV Formats</a></li>
-        <li><a href="../../sparql-results-xml/spec/index.html">SPARQL 1.2 Query Results XML Format</a></li>
-        <li><a href="../../sparql-graph-store-protocol/spec/index.html">SPARQL 1.2 Graph Store Protocol</a></li>
-        <li><a href="../../sparql-protocol/spec/index.html">SPARQL 1.2 Protocol</a></li>
-      </ul>
-    </section>
-    
-<!-- BODY -->
+    <!-- BODY -->
     <section id="introduction">
       <h2>Introduction</h2>
-      <p>RDF is a directed, labeled graph data format for representing information in the Web. RDF is often used to represent, among other things, personal information, social networks, metadata
-      about digital artifacts, as well as to provide a means of integration over disparate sources of information. This specification defines the syntax and semantics of the SPARQL query language for
-      RDF.</p>
-      <p>The SPARQL query language for RDF is designed to meet the use cases and requirements identified by the RDF Data Access Working Group in <a href="http://www.w3.org/TR/rdf-dawg-uc/" class=
-      "inform">RDF Data Access Use Cases and Requirements</a> [<a href="#UCNR">UCNR</a>] and <a href="http://www.w3.org/TR/sparql-features/" class="inform">SPARQL New Features and Rationale</a>
-      [<a href="#UCNR2">UCNR2</a>].</p>
+      <p>
+        RDF is a directed, labeled graph data format for representing information in the Web. RDF is
+        often used to represent, among other things, personal information, social networks, metadata
+        about digital artifacts, as well as to provide a means of integration over disparate sources of
+        information. This specification defines the syntax and semantics of the SPARQL query language
+        for RDF.
+      </p>
+      <p>
+        The SPARQL query language for RDF is designed to meet the use cases and requirements
+        identified by the RDF Data Access Working Group in [[[?UCNR]]] [[?UCNR]] and [[[?UCNR2]]]
```

Comment:
Note that ReSpec typically figures out if references are normative or informative depending on if they're in a normative/informative section. You can, of course, override this using `[[?UCNR]]` or `[[!UCNR]]`. Not wrong to do it, but just FYI.

Reply:
The `?` came from your PR. i was wondering what it did!

Reply:
I think the original reference was explicitly informative, so I probably just replicated that. It’s not worth scouring the spec to save a couple of characters, unless informative references should be normative or visa-versa.